cpu/ppc64_cpu_test.py

```diff
@@ -52,6 +53,9 @@ def setUp(self):
         self.smt_values = {1: "off"}
         self.key = 0
         self.value = ""
+        self.max_smt_value = 4
+        if cpu.get_cpu_arch().lower() == 'power8':
+            self.max_smt_value = 8
```

Comment:
What I mean of 'next cpu architecture' is for Power9. If this test case is run on Power9 it will take max_smt_value as '4'.